src/App.tsx

```diff
@@ -24,51 +70,28 @@ export const App: React.FC = () => {
 
           <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
             <div className="dropdown-content">
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Bernard Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Antone Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Petronella de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Hercke</p>
-              </div>
+              {filteredPeople.length === 0 ? (
+                <div className="dropdown-item" data-cy="no-suggestions-message">
+                  <p className="has-text-danger">No matching suggestions</p>
+                </div>
+              ) : (
+                filteredPeople.map(person => (
+                  <div
+                    key={person.slug}
```

Comment:
Generating keys using `person.slug` might not be reliable if `slug` is not guaranteed to be unique. Ensure that `slug` is unique for each person to avoid potential issues with React's reconciliation process. The checklist advises against generating keys on render, so consider using a unique identifier that is stable and unique for each person.